aiplatform/snippets/embeddings.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 func embedTexts(
-	apiEndpoint, project, model string, texts []string, task string) ([][]float32, error) {
+	apiEndpoint, project, model string, texts []string,
+	task string, customOutputDimensionality *int) ([][]float32, error) {
```

Comment:
From https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments, we should be using minimal arguments.

suggestion: Remove `customOutputDimensionality` as a parameter in favor of hard-coding in the function body.

Reply:
SG